kalliope_adapter.py

```diff
@@ -172,14 +174,18 @@ def repl2(matchobj):
                         .isoformat()
     ontvangen = datetime.now(tz=TIMEZONE).replace(microsecond=0).isoformat()
     inhoud = ps_uit['inhoud'] if ps_uit['inhoud'] else ""
-    dossiernummer = ps_uit['dossierNummer']
+    referentieABB = ps_uit['referentieABB']
     betreft = ps_uit['betreft']
     type_communicatie = ps_uit['typeCommunicatie']
     reactietermijn = "P30D"
+    dossierbehandelaar = {
```

Comment:
I am not sure there always be a dossierbehandelaar. Can you ask pieter what we want to happen if none is found?

Reply:
If not always there perhaps prefer the .get('key', {}) method

Reply:
We should (almost) always get one, and for now we'll let it fail in the cases we don't, resulting on getting an email.